local-volume/helm/provisioner/values.yaml

```diff
@@ -0,0 +1,20 @@
+common:
+  namespace: default
+  defaultHostDir: /mnt/disks
```

Comment:
this variable is not used, I think you can remove it

Reply:
done

Reply:
It's still there